java/src/main/java/com/factset/protobuf/stach/extensions/v2/ColumnOrganizedStachExtension.java

```diff
@@ -115,6 +116,13 @@ private TableData generateTable(PackageProto.Package packageObj, String primaryT
         Map<String, MetadataItemProto.MetadataItem> metadata = primaryTable.getData().getMetadata().getItemsMap();
         for (Map.Entry<String, MetadataItemProto.MetadataItem> entry : metadata.entrySet()) {
             table.getMetadata().put(entry.getKey(), StachUtilities.valueToString(entry.getValue().getValue()));
+
+            List<Value> values = new ArrayList<>();
+            List<Value> list = entry.getValue().getValue().getListValue().getValuesList();
```

Comment:
Don't we need to handle the StringValues and ListValues separately like in .Net?

Reply:
.Net was a special case because it has additional tests for "WithStruct" and "WithRowAndColSpan" which formats the metadata differently, causing there to be StringValues instead of ListValues. For example [this](https://github.com/factset/stach-extensions/blob/main/dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions.Tests/Resources/V2ColumnStachData.json#L646) is how metadata looks for a normal case  (note the brackets), and [this](https://github.com/factset/stach-extensions/blob/main/dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions.Tests/Resources/V2ColumnStachWithStruct.json#L80) is how metadata looks for one of the special cases (no brackets). Java doesn't test any special cases so therefore there are always ListValues